lib/modules/datasource/flutter/index.ts

```diff
@@ -0,0 +1,50 @@
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import type { FlutterRelease } from './types';
+
+export class FlutterDatasource extends Datasource {
+  static readonly id = 'flutter';
+
+  constructor() {
+    super(FlutterDatasource.id);
+  }
+
+  override readonly customRegistrySupport = false;
+
+  override readonly defaultRegistryUrls = ['https://storage.googleapis.com'];
+
+  override readonly caching = true;
```

Comment:
Isn't that the default?

Reply:
There's no default value set in `Datasource` ([source](https://github.com/renovatebot/renovate/blob/main/lib/modules/datasource/datasource.ts#L16)), unless `undefined` means the same as setting it to `true`?